Example/Tests/NetworkTests/UploaderTests/UploaderTests.swift

```diff
@@ -28,7 +28,7 @@ import XCTest
 class UploaderTests: NetworkBaseTest {
 
     // MARK: - Tests
-
+    // MARK: - image data
```

Comment:
These change are good, but they should be a separate PR - we can't mix completely unrelated changes (formatting, naming changes in url tests, better MARK comment) with the original intent of this PR - this will make a mess of the branch history.